osgi.enroute.examples.easse.application/osgi.enroute.examples.easse.bndrun

```diff
@@ -14,4 +14,23 @@ JPM-Command:					easse
 	osgi.identity;filter:='(osgi.identity=osgi.enroute.examples.easse.application)'
 
 -runbundles: \
-	${error;Resolve first}
+	org.apache.felix.configadmin;version='[1.8.6,1.8.7)',\
+	org.apache.felix.http.api;version='[3.0.0,3.0.1)',\
+	org.apache.felix.http.jetty;version='[3.1.0,3.1.1)',\
+	org.apache.felix.http.servlet-api;version='[1.1.2,1.1.3)',\
+	org.apache.felix.log;version='[1.0.1,1.0.2)',\
+	org.apache.felix.scr;version='[2.0.0,2.0.1)',\
+	org.eclipse.equinox.coordinator;version='[1.3.100,1.3.101)',\
+	org.eclipse.equinox.event;version='[1.3.100,1.3.101)',\
+	org.eclipse.equinox.metatype;version='[1.4.100,1.4.101)',\
+	org.osgi.service.event;version='[1.3.1,1.3.2)',\
+	org.osgi.service.metatype;version='[1.3.0,1.3.1)',\
+	osgi.enroute.dto.bndlib.provider;version='[1.0.0,1.0.1)',\
+	osgi.enroute.easse.simple.adapter;version='[1.3.2,1.3.3)',\
+	osgi.enroute.examples.easse.application;version=snapshot,\
+	osgi.enroute.executor.simple.provider;version='[1.0.0,1.0.1)',\
+	osgi.enroute.google.angular.webresource;version='[1.4.4,1.4.5)',\
+	osgi.enroute.logger.simple.provider;version='[1.2.1,1.2.2)',\
+	osgi.enroute.rest.simple.provider;version='[2.0.2,2.0.3)',\
+	osgi.enroute.twitter.bootstrap.webresource;version='[3.3.5,3.3.6)',\
```

Comment:
I think it is better to keep the error in there because this might fail with a much harder error if the setup of repos changes.